get-express-version.mjs

```diff
@@ -0,0 +1,22 @@
+import { writeFileSync } from 'node:fs'
+import { readFile } from 'node:fs/promises'
+import path from 'node:path'
+
+const NPMURL = 'https://registry.npmjs.org/express'
+
+const response = await fetch(NPMURL).then((res) => res.json())
+
+const { next, latest } = response['dist-tags']
+
+const contentFile = await readFile(
+  path.resolve(path.join('_data', 'express.yml')),
+  'utf8'
+)
+  .then((content) =>
```

Comment:
It's better to use `await` consistently instead of mixing with `.then()`.

Reply:
done, thanks